src/test/java/ai/pluggy/client/integration/helper/ItemHelper.java

```diff
@@ -22,7 +24,7 @@
   public static final String NON_EXISTING_ITEM_ID = "ab9f7a00-7d45-458b-b288-4923e18a9e69";
 
   public static final Integer PLUGGY_BANK_CONNECTOR_ID = 0;
-  public static final Integer PLUGGY_BANK_CONNECTOR_WITH_MFA_ID = 1;
+  public static final Integer PLUGGY_BANK_CONNECTOR_WITH_MFA_ID_WAITING_USER_INPUT = 3;
```

Comment:
Why `WAITING_USER_INPUT` its appended to this constant.
ConnectorID #3 is the MFA one.

Reply:
because the number 3 represent WAITING_USER_INPUT status when call getItem